en/docs/references/ballerina-expression-syntax.md

```diff
@@ -1,422 +1,495 @@
+    ```ballerina
+    60 * 5;
+    ```
 
-  ```ballerina
-  firstNumber / secondNumber
-  ```
+- Modulo 
 
-### Number Multiplication
+    You can use the  `%` operator to get the remainder of a division. 
 
-You can use the `*` operator for number multiplication. The number values can come from a literal or a variable.
+    ###### Example:
 
-#### Examples
+    To get the remainder of the division between an integer variable `n` and an integer literal `3`, you can use the following expression:
+  
+    ```ballerina
+    (n % 3) > 2
+    ```
 
-- Multiplying integer literals.
+### Value/type operators
 
-  ```ballerina
-  10 * 20
-  ```
-  
-- Multiplying decimal literals.
+- Equality 
 
-  ```ballerina
-  0.56 * 1.84
-  ```
-  
-- Multiplying a numerical variable (`firstNumber`) by a literal.
+    You can use the equality operators to check the equality of [Ballerina's](https://ballerina.io/) basic type literals. These values can come from a literal or a variable. **Note** that the resulting value from these operators is always of `boolean` type.   
+    
+    Ballerina provides the following equality operators: 
+    
+    - `==`: For deep value equality of `anydata` typed values.
+    - `===`: For reference equality. 
+    - `!= `and `!==`: For negations.
 
-  ```ballerina
-  firstNumber * 100
-  ```
-  
-- Multiplying a numerical variable (`firstNumber`) by another numerical variable (`secondNumber`).
+    ###### Examples:
 
-  ```ballerina
-  firstNumber * secondNumber
-  ```
+    To compare an integer literal `10` with an integer variable `firstNumber`, you can use the following expression:
+    
+    ```ballerina
+    10 == firstNumber
+    ```
 
-### Modulo Operation
+    To compare a string variable `username`  with  a string literal `admin`, use the following expression:  
 
-You can use the  `%` operator to get the remainder of a division. The number values can come from a literal or a variable.
+    ```ballerina
+    username != "admin" 
+    ```
 
-#### Examples
+- Value comparison
 
-- Calculating the modulus of two integer literals.
+    The following operators are available in the expression editor for value comparisons:
+    
+    - `<`: Less than
+    - `>`: Greater than
+    - `<=`: Less than or equal to
+    - `>=`: Greater than or equal to
 
-  ```ballerina
-  10 % 4
-  ```
-  
-- Calculating the modulus of a literal and a variable (`firstNumber`).
+    ###### Examples:
 
-  ```ballerina
-  firstNumber % 100 
-  ```
+    To compare the value of the integer variable `firstNumber` to the integer literal `10`, use one of the following expressions:
+
+    ```ballerina
+    firstNumber > 10
+  
+    firstNumber <= 10
+    ```
+
+    To compare the value of the string variable `greeting` to the string literal `Hi`, use the following expression:
+   
+    ```ballerina
+    greeting == "Hi"    
+    ```
+    To compare an array variable `arrayWithTwoElements` with an array literal `arrayWithOneElement`, use the following expression:
```

Comment:
`arrayWithOneElement` is also a variable here?

Reply:
Yes. Corrected.